src/components/sidebar/sidebar.stories.mdx

```diff
@@ -87,6 +88,30 @@ with all the UI pass `enableBackgroundUI={ true }` to the component
   </Story>
 </Canvas>
 
+### Custom padding around content
+
+<Canvas>
+  <Story name="custom padding around content">
+    {() => {
+      const [isOpen, setIsOpen] = useState(isOpenForChromatic);
+      return (
+        <>
+          <Button onClick={() => setIsOpen(true)}>Open sidebar</Button>
+          <Sidebar open={isOpen} onCancel={() => setIsOpen(false)} padding={0}>
```

Comment:
It might be better to use the `p` prop rather than `padding` here - they have the same effect (I believe, from the Styled System docs, that they are aliases of each other), but only `p` is included in our props tables, and it might be confusing to use a prop which is absent from the props table on the same page!

(I'd be happy with an alternative fix to `styled-system-props.js` in `/.storybok/utils` to include the `padding` prop as well though, I think similar changes could be made for margin etc. I assume it might have been a deliberate decision to leave the long versions of the names out though?)

Reply:
Good point, perhaps the util fix is something for us to look into in the future, but will make this change now 👍🏼

Reply:
Partially related to the prop table - I will update the proptypes for the `size` and `position` props - since I've noticed the permitted values for both props aren't being displayed for the types.

Reply:
great stuff - I noticed that too but decided it wasn't relevant to mention in this PR!